views/heatmap.py

```diff
@@ -0,0 +1,114 @@
+import dash_html_components as html
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+from datetime import timedelta
+
+from components import heatmap
+from components.config import heatmap_config
+from utils import earthquake_data
+from app import app
+from utils.dateutils import get_datetime_from_str
+from utils import session
+
+
+def get_layout(session_id):
+    """ Return the layout for a heatmap and its configurations.
+
+    Keyword arguments:
+    session_id -- ID of the current session
+    """
+    eq_data = earthquake_data.get_earthquake_data(session_id)
+
+    if eq_data is None or eq_data.data.shape == (0, 0):
+        return 'No uploaded data found'
+
+    start_date, end_date = eq_data.get_daterange()
+    filtered_data = eq_data.filter_by_dates(start_date, end_date)
+
+    default_x = filtered_data.get_magnitudes()
+    default_y = filtered_data.get_latitudes()
```

Comment:
Would depth be a better default y-axis, as it's also used in the scatter plot?

Reply:
Yes, my intention is to have the default y-axis as depth, but I had a bit of trouble with one of the depth columns in one of the datasets earlier, but I will change that back. The feature names should be possible to implement, but the ranges might be trickier.